### PR TITLE
BETA: Register cisainternet1.is-a.dev

### PR DESCRIPTION
**Pull request by CISAFounder**
Added `cisainternet1.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/cisainternet1.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CISAFounder",
+        "email": "calebinternet@outlook.com"
+    },
+    "record": {
+        "URL": "cisainternet.com"
```
Comment (CuteDog5695) [inline domains/cisainternet1.json:7]: URL needs protocol (also site redirects to IONOS saying that there isnt a website connected yet)

File: domains/caleba.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CISAFounder",
+        "email": "calebinternet@outlook.com"
+    },
+    "record": {
+        "URL": "cisainternet.com"
```
Comment (CuteDog5695) [inline domains/caleba.json:7]: URL needs protocol (also site redirects to IONOS saying that there isnt a website connected yet)

File: domains/cisainternet.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "CISAFounder",
+        "email": "calebinternet@outlook.com"
+    },
+    "record": {
+        "CNAME": "cisainternet.com"
```
Comment (CuteDog5695) [inline domains/cisainternet.json:7]: redirects to IONOS saying that website isnt connected yet

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]

File: domains/cisainternet1.json (hunk 1 above)
Comment (CISAFounder) [reply domains/cisainternet1.json:7]: What do you mean by prtocol

Comment (CuteDog5695) [reply domains/cisainternet1.json:7]: Like https://

**Pull request closed without merging**